packages/std/src/query.rs

```diff
@@ -13,6 +13,16 @@ pub enum QueryRequest<C: CustomQuery> {
     Bank(BankQuery),
     Custom(C),
     Staking(StakingQuery),
+    /// A Stargate message encoded the same way as a protobof [Any](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto).
+    /// The format is defined in [ADR-21](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-021-protobuf-query-encoding.md)
```

Comment:
Copy/pase bug?

Reply:
It is ADR-21 not ADR-20 (above).
I did copy paste, but thought I made the change to the link

Reply:
Okay, but this is not a "message", right? "A Stargate message encoded the same way as a protobof Any"